Makefile

```diff
@@ -24,6 +25,11 @@ else    # $(MSBUILD) != 1
 _SLN_BUILD  = MSBUILD="$(MSBUILD)" tools/scripts/xabuild
 endif   # $(USE_MSBUILD) == 1
 
+ifneq ($(ANDROID_API),)
+BUILD_FRAMEWORK_VERSION=$(shell echo "$(ALL_FRAMEWORKS)"|tr -s " "|cut -d " " -s -f $(ANDROID_API))
```

Comment:
Instead of using `tr`, this should instead be `$(word $(API_LEVEL), $(ALL_FRAMEWORKS))`. (Using `$(API_LEVEL)` instead of `$(ANDROID_API)`, as [suggested earlier](https://github.com/xamarin/xamarin-android/pull/1353#issuecomment-369454979).